.github/workflows/pages.yml

```diff
@@ -0,0 +1,90 @@
+name: Configurable Systems Pages
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
```

Comment:
Is there a specific reason why we use Ubuntu 20.04? Couldn't we also use Ubuntu 22.04?

Reply:
@s9latimm If it works, we could also move to 22.04